stats.lua

```diff
@@ -2,76 +2,76 @@ local x = {}
 x["Version"] = "|cFFFAFA44Icy-Veins Stat Priorities:|cFF00EA00 09.01.2023|r"

```

Comment:
Oooops forgot to change the date here. Be sure to fix when you merge. I'm not sure if there's an easier way for me to change this without making a new pull request.